src/renderer/components/dialogs/Settings-ExperimentalFeatures.tsx

```diff
@@ -87,10 +96,56 @@ export function SettingsExperimentalFeatures({
       >
         {tx('videochat')}
       </SettingsSelector>
+      <DebugWebxdcSwitch />
     </>
   )
 }
 
+function DebugWebxdcSwitch(props: {}) {
+  const tx = window.static_translate
+
+  const [debugMsgId, setDebugMsgId] = useState(0)
+
+  const refresh = useCallback(() => {
+    DeltaBackend.call('settings.getConfig', 'debug_logging').then(val =>
+      setDebugMsgId(Number(val))
+    )
+  }, [])
+
+  useEffect(() => {
+    refresh()
+  }, [refresh])
+
+  console.log({ debugMsgId })
+  const isActive = !isNaN(debugMsgId) && debugMsgId > 0
+  return (
+    <div>
+      <DeltaSwitch2
+        label={tx('debug_logging')}
+        description={tx('debug_logging_description')}
```

Comment:
Is `tx` for translating? Because, the names are probably going to be pref_debug_logging and pref_debug_logging_explain for consistency with other preferences.

See https://github.com/deltachat/deltachat-android/pull/2292/files#diff-ac392d3094c70ee4fda24e1a3a80f912c9f14d01fbf4ed7a774b8ba09c301188

Reply:
yes tx is for translation, If you want you can change those strings too or comment a suggestion

Reply:
```suggestion
        label={tx('pref_debug_logging')}
        description={tx('pref_debug_logging_explain')}
```